src/query/execute_schema.c

```diff
@@ -379,6 +379,8 @@ static int do_alter_index_status (PARSER_CONTEXT * parser, const PT_NODE * state
 
 int ib_thread_count = 0;
 
+#define MAX_QUERY_STRING_LENGTH 4096
```

Comment:
How about calling it MAX_VIEW_QUERY_STRING_LENGTH.
I think the meaning may be misunderstood.